packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -21,19 +21,20 @@ import 'scrollbar.dart';
 const TextStyle _kCupertinoDialogTitleStyle = TextStyle(
   fontFamily: '.SF UI Display',
   inherit: false,
-  fontSize: 18.0,
+  fontSize: 17.0,
```

Comment:
The font size change here (and the one below) looks good to me. It matches what I saw in Xcode too. I'm not sure how the rest of them (height and letterSpacing) are derived. Could you document where these values came from (similar to https://github.com/flutter/flutter/issues/21302)? This will help other maintainers when they revise these values.

Reply:
Sure! I've added code comments.

Reply:
You are saying the values are from the said plugin, but you had to make some adjustments?

Reply:
Yup, that's right. Some rounding off and visual comparison is needed.
What I got from the plugin:
* Title
```dart
fontSize: 17,
letterSpacing: -0.544, // Rounded up to -0.5
height: 1.2941176470588236, // Rounded up to 1.3
```
* Content
```dart
fontSize: 13,
letterSpacing: -0.10400000000000001,
height: 1.2307692307692308,
```
The title `TextStyle` is okay, but content `TextStyle` needed adjustments.
When rounded `letterSpacing` and `height` of content to -0.1 and 1.2, the result looks like below.
| Before | Native Dialog | Before adjustments | After adjustments |
|---|---|---|---|
| <img src="https://user-images.githubusercontent.com/33684401/93506339-1780a800-f957-11ea-9f45-f2226c7ab23a.png"> | <img src="https://user-images.githubusercontent.com/33684401/91070565-d952da00-e671-11ea-9364-8048a859382e.png"> | <img src="https://user-images.githubusercontent.com/33684401/93506241-f7e97f80-f956-11ea-969e-eb3a753f8cef.png"> |<img src="https://user-images.githubusercontent.com/33684401/91070605-e8d22300-e671-11ea-92d8-ca56685ab3ca.png"> |